enhancements/windows-containers/health-management.md

```diff
@@ -66,18 +66,26 @@ configuration of a Windows instance, to ensure that it can become a worker Node.
 
 To accomplish the goals listed in this enhancement, I am proposing that WMCB be converted into a daemon,
 Windows Instance Config Daemon (WICD). WICD will have the following responsibilities:
-* WICD maintains current responsibilities of bootstrapping kubelet through the `bootstrap` command.
+* As part of the `bootstrap` phase, WICD will configure the [containerd](container-runtime-containerd.md) runtime and 
+  start it as a Windows service, if its configuration is detailed in the services ConfigMap.
+* WICD will also maintain current responsibilities of starting kubelet as a service through the `bootstrap` command. 
+  WICD will fully configure kubelet in one shot, as everything required will be available in the services ConfigMap.
 * Using the configuration provided by a ConfigMap created by WMCO, WICD maintains the state of Windows Services on the 
   instance.
 * WICD reverts all changes made to an instance when run with the `cleanup` command, and also deletes the Node
   object.
 
 To enable this, WMCO will have these changes in responsibility:
 * WMCO will no longer configure the Windows services on Windows instances, except for WICD.
+* WMCO will take over the parsing of the worker ignition file. Note that WMCO will now be able to provide all the 
```

Comment:
While this might be needed I don't understand the reasoning behind doing so. So more explanation is needed regarding the why. All that is mentioned here is the how.

Reply:
The idea behind this is that kubelet no longer needs to be restarted with its initial config being overwritten with new CNI options. Since WMCO transfers (and deletes when deconfiguring) all other files onto the node, it should extract the needed info from ignition and handle the transport onto instances. We thought WICD should not be responsible for transferring anything -- it should just manage the state of already present binaries and files.

Reply:
>We thought WICD should not be responsible for transferring anything

WICD is on the instance. How is it ever going to transfer anything?

Reply:
The reason that needs to be captured here is that WMCO is the maintainer of the services ConfigMap. WMCO must provide the expected state of the kubelet service, and must parse the ignition file in order to do so.